core/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/LongTerms.java

```diff
@@ -126,26 +99,36 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             builder.endObject();
             return builder;
         }
+
+        public DoubleTerms.Bucket convertToDouble() {
```

Comment:
I'll move it back. It felt right at the time but I can't defend the decision in the morning.